poglink/cogs/rates.py

```diff
@@ -115,15 +116,15 @@ async def on_ready(self):
                 logger.info(f"Retrieving current rates at {url}")
                 try:
                     rates = await self.get_current_rates(url)
-                    await asyncio.sleep(5)
+                    await asyncio.sleep(MIN_POLLING_DELAY)
```

Comment:
Hm, but it's still hard-coded; it's just hard coded to be the constant `MIN_POLLING_DELAY` instead of the literal `5`.